encord/client.py

```diff
@@ -612,20 +613,31 @@ def add_users(self, user_emails: List[str], user_role: ProjectUserRole) -> List[
 
         return list(map(lambda user: ProjectUser.from_dict(user), users))
 
-    def copy_project(self, copy_datasets=False, copy_collaborators=False, copy_models=False) -> str:
+    def copy_project(
+        self,
+        copy_datasets=False,
+        copy_collaborators=False,
+        copy_models=False,
+        copy_labels=False,
```

Comment:
We started to introduce keyword only arguments in places. I think having a `*,` here before the new args also makes sense